base_import_match/models/base_import.py

```diff
@@ -83,14 +83,14 @@ def _register_hook(self, cr):
     def _compute_name(self):
         """Automatic self-descriptive name for the setting records."""
         for s in self:
-            s.name = "{}: {}".format(
+            s.name = u"{}: {}".format(
                 s.model_id.display_name,
-                " + ".join(
+                u" + ".join(
                     s.field_ids.mapped(
                         lambda r: (
                             str(r.field_id.name) +
```

Comment:
Here you could have other unicode error.
![screen shot 2016-09-07 at 2 14 15 pm](https://cloud.githubusercontent.com/assets/6644187/18325080/6285ec28-7505-11e6-87b5-dbe6beefc9af.png)


What about use `r.field_id.name` without `str`?